src/AspNetCore.Lambda/HttpHandlers.fs

```diff
@@ -275,4 +291,25 @@ let htmlFile (relativeFilePath : string) =
                 ctx
                 |> (setHttpHeader "Content-Type" "text/html"
                 >>= setBodyAsString html)
-        }
\ No newline at end of file
+        }
+
+let private handlerWithRootedPath (path:string) (handler : HttpHandler) = 
```

Comment:
Let's move this up to the other private methods and move the `subRoute` and `subRouteCi` below the other route methods please and add some spaces in `(path : string)`. Sorry I know I am being anal here :)